main/src/cgeo/geocaching/connector/ec/ECConnector.java

```diff
@@ -86,6 +86,15 @@ public String getNameAbbreviated() {
         return "EC";
     }
 
+    @Override
+    @NonNull
+    public String getTestUrl() {
```

Comment:
The test data should be in the test code.

Reply:
That's the way the `IConnector` interface is implemented currently (and used by the `GCConnector` as well).

But anyway, it's obsolete now, as I I've removed the addition of an individual testUrl for the EC connector due to the change requested by @Lineflyer.